mods/cnc/rules/campaign-maprules.yaml

```diff
@@ -15,13 +15,6 @@ World:
 	MapOptions:
 		ShortGameCheckboxLocked: True
 		ShortGameCheckboxEnabled: False
-	ScriptLobbyDropdown@difficulty:
```

Comment:
Not all missions define this in their `rules.yaml` right? That will mean they crash because the global script tries to find this dropdown.

Reply:
We need this removed to fix the menu order, so I took it out of the global script and added the line there to every mission that needs it.

Reply:
Can we add the trait to all the missions which don't have it yet instead? Why is the menu order affected by this in the first place?

Reply:
Not sure why but I'm 100% sure it is the fix. I noticed TD was the only mod that had the dropdown in campaign-rules. RA missions without difficulty levels are done the way I have it here.

Reply:
> Why is the menu order affected by this in the first place?

IIRC, we still have some subtle yaml merge issues with map rules, I suspect those are to blame. The order probably depends on which yaml file is merged first. I don't think we should hold up the PR over this.

Reply:
@abcdefg30 are you okay with this? The PR looks fine to merge otherwise, but I don't want to do that if we're missing a better fix.

Reply:
I find it annoying that we do a bunch of changes here we know we need to revert down the line, but can live with it.

Reply:
The problem is going to be that this default defines Normal first, and we then merge the additions below it.

Would a compromise where we define everything except the Values here be to weird?

Reply:
Imo that would be ok. Most of the values here (e.g. `Default: normal`) are shared between all missions already anyway.

Reply:
Restored the ID and Label definitions.

Reply:
Hmm. Values are apparently required from `ScriptLobbyDropdown@difficulty`.